.github/workflows/kind-e2e.yaml

```diff
@@ -202,98 +187,11 @@ jobs:
-        kind: Cluster
-
-        # Configure registry for KinD.
-        containerdConfigPatches:
-        - |-
-          [plugins."io.containerd.grpc.v1.cri".registry.mirrors."$REGISTRY_NAME:$REGISTRY_PORT"]
-            endpoint = ["http://$REGISTRY_NAME:$REGISTRY_PORT"]
-          [plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
-            endpoint = ["https://mirror.gcr.io"]
-
-        # This is needed in order to support projected volumes with service account tokens.
-        # See: https://kubernetes.slack.com/archives/CEKK1KTN2/p1600268272383600
-        kubeadmConfigPatches:
-          - |
-            kind: ClusterConfiguration
-            metadata:
-              name: config
-            apiServer:
-              extraArgs:
-                "service-account-issuer": "kubernetes.default.svc"
-                "service-account-signing-key-file": "/etc/kubernetes/pki/sa.key"
-            networking:
-              dnsDomain: "${CLUSTER_DOMAIN}"
-
-        nodes:
-        - role: control-plane
-          image: kindest/node:${{ matrix.k8s-version }}@${{ matrix.kind-image-sha }}
-          extraMounts:
-          - containerPath: /var/lib/etcd
-            hostPath: /tmp/etcd
-        - role: worker
-          image: kindest/node:${{ matrix.k8s-version }}@${{ matrix.kind-image-sha }}
-        - role: worker
-          image: kindest/node:${{ matrix.k8s-version }}@${{ matrix.kind-image-sha }}
-        - role: worker
-          image: kindest/node:${{ matrix.k8s-version }}@${{ matrix.kind-image-sha }}
-        - role: worker
-          image: kindest/node:${{ matrix.k8s-version }}@${{ matrix.kind-image-sha }}
-        EOF
-
-        kind create cluster --config kind.yaml --wait 5m
-
-    - name: Install metallb
-      shell: bash
-      run: |
-        # Install metallb
-        # disable the webhook-mode because there is a bug:https://github.com/metallb/metallb/issues/1597
-        # this webhook only checks crd, it has not effects to out e2e tests
-        curl https://raw.githubusercontent.com/metallb/metallb/v0.13.5/config/manifests/metallb-native.yaml -k | \
-         sed '0,/args:/s//args:\n        - --webhook-mode=disabled/' | \
-         sed '/apiVersion: admissionregistration/,$d' | \
-         kubectl apply -f -
-
-        # Add Layer 2 config
-        network=$(docker network inspect kind -f "{{(index .IPAM.Config 0).Subnet}}" | cut -d '.' -f1,2)
-        cat <<EOF | kubectl apply -f -
-        apiVersion: metallb.io/v1beta1
-        kind: IPAddressPool
-        metadata:
-          name: first-pool
-          namespace: metallb-system
-        spec:
-          addresses:
-          - $network.255.1-$network.255.250
-        ---
-        apiVersion: metallb.io/v1beta1
-        kind: L2Advertisement
-        metadata:
-          name: example
-          namespace: metallb-system
-        EOF
-
     - name: Setup local registry
       run: |
+       docker stop $REGISTRY_NAME
```

Comment:
This PR is still WIP so you may change here, but these `docker stop` and `docker rm` are necessary? If so can you leave a code comment?

Reply:
Thank you! Please feel free to remove WIP if you are ready.
It seems lint error is happening https://github.com/knative/serving/actions/runs/5204469276/jobs/9388720626?pr=14018 though